modules/ocf/files/packages/cups/convert_failure

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+"""A script to send an email to the root mailing list upon failure of
+the convert program, when used in the raster-filter script, a
+Tea4CUPS filter.
+
+Jobs printed from one of the desktops, or from supernova or tsunami,
+first go through raster-filter, which calls convert to reduce the
+density of any PDFs, essentially rasterizing them. If convert happens
+to fail, raster-filter will pipe stderr from convert to this script,
+which takes in stdin and includes it in an email to root@ocf. Thus,
+this script should not be called casually by regular users!
+
+Based off the enforcer script for whiteout, and the
+email_ldapsync_problem script
+"""
+import socket
+import sys
+from textwrap import dedent
+
+from ocflib.misc import mail
+
+ERROR_MSG = dedent("""\
+        raster-filter encountered the following error while running convert:
+
+        {stdin}
+
+
+        ====
+        Hostname: {hostname}
+""")
+
+
+def main():
+    if sys.stdin.isatty():
+        sys.stderr.write('script should not be called interactively. Aborting...')
+    else:
+        error_message = ERROR_MSG.format(stdin=sys.stdin.read(),
+                                         hostname=socket.getfqdn()
+                                         )
```

Comment:
Did you run `pre-commit`? Formatting is weird

Reply:
Done, see commit https://github.com/ocf/puppet/pull/684/commits/f319632133609bca62de836b443b1f66c404b293